packages/atomic/src/components/search/generated-answer/atomic-generated-answer/atomic-generated-answer.pcss

```diff
@@ -17,6 +32,16 @@
   background: var(--atomic-primary-background);
 }
 
+atomic-modal {
+  width: 100%;
+}
+
+.source-citations {
```

Comment:
Setting those variables like so seems like a test artefact ? Or is this really wanted ?

Shouldn't they be following other variables in use in Atomic, or rem at least ?